### ISSUE TITLE
Incorrect control type for the month and year control for fabric Date picker control 

### ISSUE THREAD

**Issue by Sandhya-aluri** [microsoft/fluentui#13919]
### Environment Information

- **Package version(s)**: "office-ui-fabric-react": "^7.94.1"
- **Browser and OS versions**: chromium Edge , Windows 10

When focus is on the month / year in date picker control. It is not reading as month/year. For example when the focus is on Jan, as it is clickable it should be read as button. but reading as just Jan.
The control type in AI4D is mentioned as Item instead of button.

The control types for months/years should be button.

#### Actual behavior:

The given control type for the month/year in Date picker control is "Item" but it is working as "Button".

#### Expected behavior:

The control type should be a button for the month and year as they are clickable.
<img width="670" alt="Capture1" src="https://user-images.githubusercontent.com/48669559/86470063-458f1c80-bd58-11ea-89ca-027d84c0535f.PNG">

**Comment by lorejoh12** [comment]
This is sort of by design, in that we made a tradeoff of info.

It's due to the role=gridcell that we put on the buttons in order to have the row/column information read out for navigation purposes. If we change the role back to "button", we lose that information. We are open to suggestions on alternative solutions, but have not found one yet that gives all the information we want.

**Comment by lorejoh12** [comment]
closing as by design, please feel free to reopen if you have a suggestion for more correct semantics that will capture all the information we want

**Comment by Sandhya-aluri** [comment]
lorejoh12  : As per our requirement we need this to be fixed for accessibility compliance. is there any possibility to reopen and address this issue?

**Comment by lorejoh12** [comment]
There is. If you have a suggestion for how to satisfy both requirements, I'm happy to either make the correct fix or review a PR.
We need row/column information to be read out when arrowing around the grid, due to the grid navigation structure. Changing role to "button" makes that information not get read. So, the fix can't just be "change role to 'button' role", because then we'll have another accessibility escalation for that information missing. The last accessibility escalation about missing navigation information was what made us make this change, and we determined the navigation information to be more important than the button information at that time, despite what the accessibility tool says.

Any other suggestions though, or an explanation (and ideally some links to documentation, which I haven't been able to find myself) of why the semantic information that it's a button is more important than the navigation information, will let us open this again.
